components/monitoring/README.md

```diff
@@ -0,0 +1,113 @@
+###On boarding to services to Monitoring
+
+#1. Metrics exporter services
+a. The intended service should export the metrics from the application so that prometheus is able to understand it. 
+b. For refernce https://prometheus.io/docs/instrumenting/writing_exporters/
+c. Exported port, service, route should be accessible to prometheus service.
+[Here](https://github.com/redhat-appstudio/service-provider-integration-operator/blob/main/config/rbac/auth_proxy_service.yaml) is example for the spi-system
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: controller-manager-metrics-service
+  namespace: system
+spec:
+  ports:
+  - name: metrics
+    port: 8443
+    protocol: TCP
+    targetPort: https
+  selector:
+    control-plane: controller-manager
+    app.kubernetes.io/name: service-provider-integration-operator
+```
+
+#2. Service monitors
+a.
+i. If service monitor is for prometheus it self
+* Add service monitor declaration for scrapping
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  namespace: appstudio-workload-monitoring
+  name: prometheus
+  labels:
+    prometheus: appstudio-workload
+spec:
+  endpoints:
+  - bearerTokenSecret:
+      key: ""
+    interval: 15s
+    path: /metrics
+    port: oauth2-proxy
+    scheme: HTTPS
+    tlsConfig:
+      caFile: /etc/prometheus/tls/tls.crt # volume mounted in the `prometheus` container of the prometheus pods
+      serverName: "prometheus-oauth2.appstudio-workload-monitoring.svc"
+  namespaceSelector:
+    matchNames:
+    - appstudio-workload-monitoring
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: monitoring-workload-in-cluster
+```
+* [Here](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/monitoring/prometheus/base/prometheus-servicemonitors.yaml) is an example servicemonitor for prometheus itself
+ii. If the service is for getting other components added to prometheus monitoring
+* Add a Sevice monitoring decalration for scrapping
```

Comment:
This is rendering strangely because the Markdown is not correct, but I think you could combine these two lines (58 and 59) and change scrapping -> scraping, plus decalration -> declaration.

Reply:
I want to make it as declaration of servicemonitor for scarping.